doc/migrations/v0.45-v0.46.md

```diff
@@ -0,0 +1,119 @@
+# Migrating to libp2p@46 <!-- omit in toc -->
+
+A migration guide for refactoring your application code from libp2p v__ to v__.
+
+## Table of Contents <!-- omit in toc -->
+
+- [Graceful stream closing](#graceful-stream-closing)
+- [Stream/Connection stat properties](#streamconnection-stat-properties)
+- [Interface module consolidation](#interface-module-consolidation)
+
+## Graceful stream closing
+
+Streams can either be closed gracefully, where we wait for any unsent data to be sent, or aborted in which case any unsent data is discarded and a reset message is sent, notifying the remote of the abnormal termination.
```

Comment:
If a stream is aborted, and the `.reset` method has been removed, why is it that a _reset_ message is sent as opposed to an _abort_ message?

Reply:
There's no such thing as an abort message.

Calling `.abort` will cause a reset message to be sent to the remote.

Receiving a reset message from the remote will cause the stream to throw.

Reply:
That's good to know, whenever working with yamux I often wondered what was the distinction between the [two](https://github.com/ChainSafe/js-libp2p-yamux/blob/dc55dcf4972c4805cebea4c3a4c18eafc9d17adc/src/constants.ts#L26-L27) albeit the spec rightly only makes reference to a stream reset.